components/contactUs.tsx

```diff
@@ -0,0 +1,139 @@
+import React, { useState, useRef, useEffect } from "react";
+import { RenderMsg } from "./renderMsg";
+
+type FormState = {
+  name: string;
+  email: string;
+  message: string;
+};
+
+export const ContactFrom = () => {
```

Comment:
It's a general rule of thumb that the file should be named the same as the component it contains. In this case: `ContactFrom.tsx`